pkg/lib/serverTime.js

```diff
@@ -531,7 +531,8 @@ function ChangeSystimeBody({ state, errors, change }) {
                                         weekStart={timeformat.firstDayOfWeek()}
                                         placeholder={timeformat.dateShortFormat()}
                                         onChange={d => change("manual_date", d)}
-                                        value={manual_date} />
+                                        value={manual_date}
+                                        appendTo={() => document.body} />
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17826-20221018-195623-2ba05daf-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)